backend/app-orchestrator-service/api/src/main/kotlin/api/Compute.kt

```diff
@@ -298,4 +315,8 @@ ${req("6", true, JobsControl::update, "Proceed `FOO123` to `FAILURE`")}
             CommonErrorMessage>("openInteractiveSession") {
         httpUpdate(baseContext, "interactiveSession", roles = Roles.PRIVILEGED)
     }
+
+    val utilization = call<UtilizationRequest, UtilizationResponse, CommonErrorMessage>("utilization") {
+        httpUtilization(baseContext)
```

Comment:
The other `httpXXX` methods are used only to define conventions. This utilization endpoint isn't so special that we need to invent a new one :wink: https://github.com/SDU-eScience/UCloud/issues/1934

```suggestion
        httpRetrieve(baseContext, "utilization")
```